### PR TITLE
Corrected Spellings

### PR DESCRIPTION
**Pull request by chinmayPro24**
I Corrected Spelling Mistakes In ReadMe File

### CODE REVIEW

**Comment by chinmayPro24** [comment]
Corrected Spellings

**Comment by AnkushSinghGandhi** [comment]
you don't need to add "Example--" again 
Screenshot from 2021-09-14 17-45-37

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ diverse, inclusive, and healthy community.
 
 ## Our Standards
 
-Examples of behavior that contributes to a positive environment for our
+Examples of behaviour that contributes to a positive environment for our
```
Comment (dipakbari4) [inline CODE_OF_CONDUCT.md:17]: This isn't a spelling mistake. It's US English and you're following UK English. Both are correct and can be used in the preferred language style.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ community include:
 * Focusing on what is best not just for us as individuals, but for the
   overall community
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behaviour include:
```
Comment (dipakbari4) [inline CODE_OF_CONDUCT.md:28]: This isn't a spelling mistake. It's US English and you're following UK English. Both are correct and can be used in the preferred language style.

**Review by dipakbari4 — APPROVED** [review]
Changes approved

**Review by dipakbari4 — APPROVED** [review]

**Comment by vartikaabc** [comment]
Nice

**Comment by AnkushSinghGandhi** [comment]
closing this pull request  as there is no activity feel free to create a new PR

**Pull request closed without merging**